config/routes.rb

```diff
@@ -0,0 +1,17 @@
+Rails.application.routes.draw do
+  resources :foods
+  devise_for :users
+
+  get 'home/index'
+  root :to => "home#index"
```

Comment:
- [ ] I would recommend using root path with `recipes index` or if you want to show the `home#index`, please write something relevant on the home page so that when users `sign-in` they see some relevant test instead of `Home#index
Find me in app/views/home/index.html.erb`.